### DISCUSSION TITLE
GP11 use as a dashcam

### DISCUSSION THREAD

**Discussion by remltr12** [community/community#138079] [Other Feature Feedback, Questions, & Ideas]
I have just a rudimentary understanding of the code used in this environment so don't hate, at least at the start. I have a GP11 that I want to use as a dashcam. I will have usb power. No battery due to overheating. Settings that I need: unlocked as I will be turning the camera upside down. Video looping since I will only need the last chapter or two if something happens and so I won't have to keep formatting the sd card when it becomes full. Quik Capture on power. 1080P 60 W. Super smooth. I understand that there is no way to save video when power is lost but the fix on the next start will recover it. Hopefully I have covered all the details.

**Comment by LiteBrite82** [comment]
Hi @remltr12 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
